src/credentials/CHIPCert.h

```diff
@@ -467,6 +476,11 @@ class DLL_EXPORT ChipCertificateSet
      **/
     CHIP_ERROR LoadCerts(chip::TLV::TLVReader & reader, BitFlags<CertDecodeFlags> decodeFlags);
 
+    /**
+     * @brief Release last certificate data in the set.
```

Comment:
The name already says this. Why is this useful? Is there a possibility of composing other existing methods to do the same without a new method?